tools/convert/rs-convert.cpp

```diff
@@ -37,6 +38,7 @@ int main(int argc, char** argv) try
     ValueArg<string> outputFilenameBin("b", "output-bin", "output BIN (depth matrix) file(s) path", false, "", "bin-path");
     SwitchArg switchDepth("d", "depth", "convert depth frames (default - all supported)", false);
     SwitchArg switchColor("c", "color", "convert color frames (default - all supported)", false);
+    SwitchArg switchTextOutput( "T", "output-text", "output text to stdout", false );
```

Comment:
Need to update the tool readme file right?
https://github.com/IntelRealSense/librealsense/tree/master/tools/convert#readme

Reply:
Right, good catch -- added